max_translational_distance: 0.2
max_rotational_distance: 0
min_entropy_ratio: 0.6
min_eq_sys_constraint_ratio: 0.3
constraint_search_radius: 5.0
constraint_min_entropy_ratio_coarse: 0.03
constraint_min_entropy_ratio_fine: 0.6
constraint_min_eq_sys_constraint_ratio: 0.3
graph_opt_dense_graph: false
graph_opt_iterations: 50
graph_opt_min_distance: 0
graph_opt_remove_outliers: true
graph_opt_outlier_weight_threshold: 0.1
graph_opt_final: true
graph_opt_final_dense_graph: true
graph_opt_final_remove_outliers: true
graph_opt_final_outlier_weight_threshold: 0.1
graph_opt_final_iterations: 1000
graph_opt_robust: true
use_multithreading: true
